components/brave_shields/browser/ad_block_service.cc

```diff
@@ -106,6 +108,69 @@ bool AdBlockService::ShouldStartRequest(
   return true;
 }
 
+base::Optional<base::Value> AdBlockService::UrlCosmeticResources(
```

Comment:
need to add headers for base/optional and base/values

Reply:
added in the header